src/app/proposals/create/CreateBuilderProposalForm.tsx

```diff
@@ -47,11 +48,7 @@ export const CreateBuilderProposalForm: FC = () => {
   const router = useRouter()
   useVotingPowerRedirect()
   const { setMessage } = useAlertContext()
-  const { onCreateBuilderWhitelistProposal, isPublishing, error } = useCreateBuilderWhitelistProposal()
-  if (error) {
-    setMessage(TX_MESSAGES.proposal.error)
-    console.error('🐛 Error writing to contract:', error)
-  }
+  const { onCreateBuilderWhitelistProposal, isPublishing } = useCreateBuilderWhitelistProposal()
```

Comment:
Should we handle the `transactionError`?

Reply:
@antomor Whe are currently handling all the errors in the `catch` statement. This includes custom errors thrown inside `onCreateBuilderWhitelistProposal` function, and transaction errors thrown by wagmi . To avoid handling error 2 times, all the errors are handled inside the catch, and the `transactionError` handling was deleted.

Reply:
I thought `transactionError` wouldn't have been thrown 👍🏼